File: Gruppe3/MailController.cs
using System;
using System.Net.Mail;

namespace Gruppe3
{
    class MailController
    {
        public static SmtpClient CreateConnection(string username, string password, string host = "smtp.gmail.com", int port = 587)
        {
            SmtpClient mailClient = new SmtpClient(host);
            mailClient.Port = port;
            mailClient.Credentials = new System.Net.NetworkCredential(username, password);
            return mailClient;
        }

        public static MailMessage CreateMail(string username, string toMail, string subject, string body = "")
        {
            MailMessage mail = new MailMessage();
            mail.From = new MailAddress(username);
            mail.To.Add(toMail);
            mail.Subject = subject;
            mail.Body = body;
            return mail;
        }
    }
}

File: LaravelBackend/app/Http/Controllers/MailController.php
<?php
namespace App\Http\Controllers;

use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\SMTP;
use PHPMailer\PHPMailer\Exception;

require __DIR__.'/../../../vendor/autoload.php';

class MailController
{
    public function NotifyUser(){
        $lektioner = \App\Lektion::all();
        dd($lektioner);
    }
    public function SendMail($user, $class)
    {
        $mail = new PHPMailer();
        try {
            //Server settings
            $mail->SMTPDebug = SMTP::DEBUG_SERVER;                      // Enable verbose debug output
            $mail->isSMTP();                                            // Send using SMTP
            $mail->Port       = 587;                                    // TCP port to connect to
            $mail->Host       = 'smtp.gmail.com';                    // Set the SMTP server to send through
            $mail->SMTPAuth   = true;                                   // Enable SMTP authentication
            $mail->Username   = env('MAIL_USERNAME');                     // SMTP username
            $mail->Password   = env('MAIL_PASSWORD');                               // SMTP password
            $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;         // Enable TLS encryption; `PHPMailer::ENCRYPTION_SMTPS` also accepted
        
            //Recipients
            $mail->setFrom(env('MAIL_USERNAME'), 'SKOLESKEMA');
            $mail->addAddress('mollevip@gmail.com', 'Joe User');     // Add a recipient
            // Content
            $mail->isHTML(true);                                  // Set email format to HTML
            $mail->Subject = 'Here is the subject';
            $mail->Body    = 'This is the HTML message body <b>in bold!</b>';
            $mail->AltBody = 'This is the body in plain text for non-HTML mail clients';
        
            $mail->send();
            echo 'Message has been sent';
        } catch (Exception $e) {
            echo "Message could not be sent. Mailer Error: {$mail->ErrorInfo}";
        }
    }
}